.gitignore

```diff
@@ -26,10 +26,13 @@ config/secrets.yml
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
 
+# Ignore pow environment settings
+.powenv 
```

Comment:
remove these 2 lines, don't need to gitignore .powenv because you don't have it installed